process/smartContract/builtInFunctions/keyValueStorage_test.go

```diff
@@ -67,3 +67,46 @@ func TestSaveKeyValue_ProcessBuiltinFunction(t *testing.T) {
 	_, err = skv.ProcessBuiltinFunction(nil, acc, vmInput)
 	require.True(t, errors.Is(err, process.ErrOperationNotPermitted))
 }
+
+func TestSaveKeyValue_ProcessBuiltinFunctionMultipleKeys(t *testing.T) {
```

Comment:
Can we perhaps add an extra unit test for the scenario `ErrNotEnoughGas`?

Reply:
done